src/coreimage.cs

```diff
@@ -2645,10 +2788,15 @@ interface CIImageProcessorKernel {
 	[iOS (8,0)]
 	[Mac (10,10)]
 	[BaseType (typeof (CIFilter))]
-	interface CIAccordionFoldTransition {
+	interface CIAccordionFoldTransition : ICIAccordionFoldTransitionProtocol {
 
+#if !XAMCORE_4_0
+		[Obsolete ("")]
```

Comment:
```suggestion
		[Obsolete ("Use 'FoldCount' instead.")]
```

Reply:
yup, like mentioned in the PR message, obsolete are not complete in the draft :)